doc/nrf/app_dev/device_guides/nrf54l/features.rst

```diff
@@ -14,10 +14,29 @@ For additional information, see the following documentation:
 * Zephyr page on the :ref:`zephyr:nrf54l15dk_nrf54l15`
 * :ref:`installation` and :ref:`configuration_and_build` documentation to install the |NCS| and learn more about its development environment.
 
+RISC-V core
+***********
+
+VPR core, named *fast lightweight peripheral processor* (FLPR), is designed to support the RISC-V instruction set, featuring key enhancements that enable efficient handling of basic and complex operations, as well as streamlined instruction processing.
+It can be used as either a standalone processor or as a helper core.
+As a helper processor, VPR specializes in managing tasks that require real-time attention or low power consumption, effectively boosting the performance of the main processor.
```

Comment:
Should this be FLPR since we went to the trouble of pointing out the name?

Could also add a paragraph break before this line, especially if Line 23 is supposed to only apply when used as a helper core.